can_tools/scrapers/base.py

```diff
@@ -1,14 +1,20 @@
 from can_tools.models import Base
+
 import os
 import pickle
+
 from abc import ABC, abstractmethod
 from pathlib import Path
 from typing import Any, Dict, List, Optional, Type
 
 import pandas as pd
+import us
 from sqlalchemy.engine.base import Engine
 
 
+ALL_STATES_PLUS_DC = us.STATES + [us.states.DC]
```

Comment:
Maybe a comment about us state versioning would be helpful

Reply:
Agreed, @cc7768 let's add a comment (similar to what you put in the PR) so we know what this was for in a couple weeks

Reply:
Good thought... I often forget why I made changes within hours (not to mention what can be forgotten over weeks/months).

I've updated the PR to reflect this.